docs/pages/guides/caching.mdx

```diff
@@ -3,8 +3,15 @@ title: "Caching"
 description: ""
 ---
 
+import StackblitzLink from "../../components/StackblitzLink";
+
 # Caching
 
+<StackblitzLink
+  entrypointFile="app/examples/new-api-cache-control/page.tsx"
+  browserPath="/examples/new-api-cache-control"
+/>
+
```

Comment:
Can we move these stackblitz links in the guides closer to where the example code is mentioned? It tends to distract from the guide itself

Reply:
In some guides we have links to the github repo for an example - I think it makes sense to have the stackblitz button right after a link to github is mentioned instead of at the top of the guide. This is typically at the end of a section, not necessarily bottom of the page